### PR TITLE
Region helper MERC-2432

### PR DESCRIPTION
**Pull request by tabayomi**
What:
This PR extends Paper by adding a location helper for content blocks. This enables access inside the helper to using all the available helpers for rendering inside Paper.

Depends on:
https://github.com/bigcommerce/paper/pull/118. 

06/14 -- Updated names to __Region__ based on new content service nomenclature

### CODE REVIEW

File: helpers/location.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function helper(paper) {
+    paper.handlebars.registerHelper('location', function (params) {
+        let locationId = params.hash.name;
+        let content = '';
+
+        if ((!paper.renderingContext) || (!paper.renderingContext.locations)) {
```
Comment (mcampa) [inline helpers/location.js]: extra parenthesis

Comment (tabayomi) [inline helpers/location.js]: not sure where you're referring..

Comment (mcampa) [inline helpers/location.js]:
```
if (!paper.renderingContext || !paper.renderingContext.locations) {
```

**Comment by tabayomi** [comment]
Merging the changes in this PR together with https://github.com/bigcommerce/paper/pull/118

Closing this.

**Pull request closed without merging**